jest.config.js

```diff
@@ -0,0 +1,9 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
+  testMatch: [
+    './src/**/__tests__/**/*.[jt]s?(x)',
```

Comment:
Only pick tests from `src` because when you ran `npm start` or `build`, `npm test` would pick the tests from `dist` also